tools/generate_init_files.sh

```
@@ -11,4 +11,4 @@ fi
 mkinit --write --black --nomods --recursive aiperf
 # Ruff check and fix just the __init__.py files, because mkinit sometimes
 # doesn't sort the imports correctly, causing infinite error loops.
-find aiperf -name '__init__.py' | xargs ruff check --fix
+find src/aiperf -name '__init__.py' | xargs ruff check --fix
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)